sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -865,7 +865,47 @@ if (contextVersion > 1) {
 	    return gl.getQueryParameter(query, pname);
 	}
     );
+
+    debug("");
+    debug("Test getFragDataLocation");
+    var baseVertShader = '' +
+    '#version 300 es\n' +
+    'uniform vec4 normal;\n' +
```

Comment:
normal's unused. Either reference it in the shader so it isn't optimized away or remove it.